router/_RPC_api.gno

```diff
@@ -71,7 +71,7 @@ func findRatios(maxHops int) []TokenRatio {
 		} else {
 			numRoutes := uint64(len(routes))
 
-			_tokenRatioX96 := u256.Zero()
+			var _tokenRatioX96 bigint
```

Comment:
After this line, `_tokenRatioX96` is infered as `u256.Uint` type, but I was wondering if there is any specific reason for using bigint?

Reply:
Blunder on my side.
If you don't mind you're welcome to change it or leave PR RFC then I'll handle it.

Reply:
https://github.com/gnoswap-labs/gnoswap/blob/e26ff5d08b36b76aefcb3aaf98489f36e6d05eef/router/_RPC_api.gno#L72-L75